composer.json

```diff
@@ -1,19 +1,41 @@
 {
-    "name": "lexpress/doctrine1",
+    "name": "friendsofsymfony1/doctrine1",
     "type": "library",
-    "description": "PHP5 Database ORM",
-    "keywords": ["orm", "database"],
+    "description": "PHP Database ORM for Symfony1. Do NOT use for new projects: please move to a newest Symfony release and Doctrine2",
```

Comment:
do use for new project 😂

Reply:
I'm just gonna use it for new project, because nobody can tell what to do!